src/python/T0/RunConfig/RunConfigAPI.py

```diff
@@ -689,11 +693,13 @@ def configureRunStream(tier0Config, run, stream, specDirectory, dqmUploadProxy):
             factory = RepackWorkloadFactory()
             wmSpec = factory.factoryWorkloadConstruction(workflowName, specArguments)
             for subscription in subscriptions:
+                logging.info("subscrition_bulk %s", subscription)
                 wmSpec.setSubscriptionInformation(**subscription)
         elif streamConfig.ProcessingStyle == "Express":
             factory = ExpressWorkloadFactory()
             wmSpec = factory.factoryWorkloadConstruction(workflowName, specArguments)
             for subscription in subscriptions:
+                logging.info("subscrition_express %s", subscription)
```

Comment:
Also typo. I'd also review the need for this log entry, and if you consider it to be necessary, then please give it a clearer format so anyone can understand what information is being given.

Reply:
Same as above